frontend/src/components/Homepage/CardsTab.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import SampleCard from "./SampleCard";
+
+function CardsTab() {
+  return (
+    <div className="w-screen flex flex-col justify-between items-center min-h-96">
+      <h3 className="w-screen text-center text-[#54EB75]">
```

Comment:
read the first PR

Reply:
div was changed into section as requested from the first PR

Reply:
one of them, not all of them... let's move on